diff --git a/node_modules/latlon-geohash/latlon-geohash.js b/node_modules/latlon-geohash/latlon-geohash.js
index cff2c89..de70e32 100644
--- a/node_modules/latlon-geohash/latlon-geohash.js
+++ b/node_modules/latlon-geohash/latlon-geohash.js
@@ -238,4 +238,4 @@ class Geohash {
 
 /* - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -  */
 
-export default Geohash;
+module.exports = Geohash;
